pandas/core/groupby/ops.py

```diff
@@ -744,7 +747,7 @@ def nkeys(self) -> int:
 
     def get_iterator(
         self, data: NDFrameT, axis: int = 0
-    ) -> Iterator[tuple[Hashable, NDFrameT]]:
+    ) -> Generator[tuple[Hashable, NDFrameT], None, None]:
```

Comment:
I believe these are equivalent, is that right?

Reply:
All generators are iterators but not all iterators have to be generators. Genrators contains "gi_frame" attribute. In this case, It practically return generator everytime but the return type is "iterator" so it raises an error in mypy because iterators doesn't necessarily have "gi_frame" attribute although practically, in our case, it always have it as it return generator.